components/Content/Videos.jsx

```diff
@@ -3,34 +3,43 @@ import Link from 'next/link';
 import videos from 'data/videos.json';
 import Video from './Video';
 
-const Videos = ({ limit }) => (
-  <section>
-    <div>
+const Videos = ({ limit }) => {
+  const videosSortedByDate = videos.sort(
```

Comment:
can we either:
- memoize this so it doesn't sort on every render
- or move it outside of the component?